### PR TITLE
Zijie lint fix for reports component Phase 11

### PR DESCRIPTION
**Pull request by iTvX**
# Description
Lint fix for Reports component, this is phase 11.

## Files changed
"All .js and .jsx files in src/components/Reports/ProjectMemberTable."

eslint test:
CleanShot 2023-10-05 at 00 50 40@2x

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/ProjectMemberTable/**/*.{js,jsx}`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

### CODE REVIEW

**Comment by emunkh404** [comment]
Hi.
I didn't find any errors or latency. I think it works as expected.

**Comment by iTvX** [comment]
> Hi. I didn't find any errors or latency. I think it works as expected.

Could you please record a video?

**Review by DavidC0126 — APPROVED** [review]
Hi I tested your PR and it looks good to me. Everything works just fine!


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/74082492/41713320-8ce3-4d11-9fbc-f0614fc2ad4c

**Review by lacnoskillz — DISMISSED** [review]
Hi Zijie,

It appears your changes work here as well. Below is a video of me testing. Note that it takes longer to load the total team report data.

[Watch the Video](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/f9be8bdc-048b-45ad-899c-f5af840e47aa)

**Review by Jiyuanxia — DISMISSED** [review]
I have reviewed and tested it. Works good!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92963742/1a4bc4ea-5c4c-422f-ae7e-e052813d6817

**Review by savio-henrique — DISMISSED** [review]
Hi @iTvX , I've tested your PR, and it all seems to be working as intended. The linting test did not show any issues and the only errors that appeared in the console I don't think are related to the lint fix!

---
## Screenshots and Video

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/83360232/d7c617fb-573f-47ee-a97f-48a75b63eae6

terminal

---

Nice work on this PR!

**Review by yi-lin-1234 — APPROVED** [review]
Hello, functionality of the ProjectMemberTable pages is the same as before. And, the eslint fix worked as expected. Great work!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92185695/6c6fb17b-809d-4cc8-88f0-0beafc5136dd

Screenshot 2023-10-25 at 10 51 05 AM

**Review by Chuehleo — DISMISSED** [review]
Everything works as expected, great job!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/122568562/f527fbc0-127c-4c38-a1b4-02c7e9e55f33

**Review by malikjahanzaib — APPROVED** [review]
Hi @iTvX, I have tested this PR locally, no lint errors, everything functioning normally. Nice job!!!
P.S. There is some delay in Total Reports but I guess it is not related to this PR. 

Screenshot:

image


Test rec:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46221334/5901e121-13d1-46b5-b97d-fcd27f2c281a

File: src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import { Stub } from '../../common/Stub';
+// eslint-disable-next-line no-unused-vars
```
Comment (EvianTan) [inline src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx:1]: Same comment as [here](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1356/files#r1384348686)

File: src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx [hunk 2]
```diff
@@ -111,7 +117,8 @@ export const ProjectMemberTable = ({ projectMembers, skip, take, handleMemberCou
         <div className="reports-table-head-cell">ID</div>
       </div>
       <div>
-        {memberFilter == 'all-time' ? (
+        {/* eslint-disable-next-line no-nested-ternary */}
```
Comment (EvianTan) [inline src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx:120]: Same as above.

File: src/components/Reports/ProjectMemberTable/index.js [hunk 3]
```diff
@@ -1 +1,4 @@
-export { ProjectMemberTable } from './ProjectMemberTable';
+import ProjectMemberTable from './ProjectMemberTable';
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (EvianTan) [inline src/components/Reports/ProjectMemberTable/index.js:3]: Same as above.

Comment (iTvX) [reply src/components/Reports/ProjectMemberTable/index.js]: fixed

File: src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx (hunk 2 above)
Comment (iTvX) [reply src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx]: fixed

File: src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx (hunk 1 above)
Comment (iTvX) [reply src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx]: fixed

**Comment by EvianTan** [comment]
image
Failed to build and cannot login via incognito window, Same as this [comment](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1356/files#r1391983106)

**Pull request closed without merging**